api/src/main/java/com/codeforcommunity/dto/site/EditStewardshipRequest.java

```diff
@@ -0,0 +1,30 @@
+package com.codeforcommunity.dto.site;
+
+import com.codeforcommunity.exceptions.HandledException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EditStewardshipRequest extends RecordStewardshipRequest {
+
+  public EditStewardshipRequest(
+      java.sql.Date date, boolean watered, boolean mulched, boolean cleaned, boolean weeded) {
```

Comment:
🔧 Since we don't import anything else named `Date` in this file, we should just be able to do `import java.sql.Date;` at the top and do `Date date` here